### ISSUE TITLE
/bin/sh: python: command not found when "dave show"

### ISSUE THREAD

**Issue by cjappl** [maxmarsc/dave#4]
(It is entirely likely I'm doing something wrong here).

System info:

```
> lldb --version
lldb-1500.0.404.7
Apple Swift version 5.10 (swiftlang-5.10.0.13 clang-1500.3.9.4)
> which lldb
/usr/bin/lldb
> sw_vers
ProductName:            macOS
ProductVersion:         14.5
BuildVersion:           23F79
```

Steps:
* Installed using the install guide
* I did not have tkinter, so I went back and installed that
* Compiled this program

```
#include <iostream>
#include <cmath>
#include <span>

using namespace std;

void fillWithSine(span<float, 1> data, float frequency, float amplitude, float phase, float sampleRate)
{
    for (int i = 0; i < data.size(); i++)
    {
        data[i] = amplitude * sin(2 * M_PI * frequency * i / sampleRate + phase);
    }
}

int main()
{
    float frequency = 440.0f;
    float amplitude = 0.5f;
    float phase = 0.0f;
    float sampleRate = 44100.0f;
    const size_t numSamples = 100;

    float data[numSamples];
    fillWithSine(span<float, 1>{data, numSamples}, frequency, amplitude, phase, sampleRate);

    for (int i = 0; i < numSamples; i++) // THIS IS LINE 26!!!
    {
        cout << data[i] << endl;
    }

    return 0;
}
```

```
lldb ./build/signals
Stop hook #1 added.
INFO: [dave] Successfully loaded
(lldb) target create "./build/signals"
Current executable set to '/Users/topher/code/leetcode_cpp/build/signals' (arm64).
(lldb) break set --file signals_main.cpp --line 26
Breakpoint 1: where = signals`main + 124 at signals_main.cpp:26:14, address = 0x0000000100003c48
```

```
(lldb) run
Process 23895 launched: '/Users/topher/code/leetcode_cpp/build/signals' (arm64)
Process 23895 stopped
* thread #1, queue = 'com.apple.main-thread', stop reason = breakpoint 1.1
    frame #0: 0x0000000100003c48 signals`main at signals_main.cpp:26:14
   23       float data[numSamples];
   24       fillWithSine(span<float, 1>{data, numSamples}, frequency, amplitude, phase, sampleRate);
   25
-> 26       for (int i = 0; i < numSamples; i++)
   27       {
   28           cout << data[i] << endl;
   29       }
Target 0: (signals) stopped.
```

Attempt to "dave show" my data buffer:
```
(lldb) dave show data
INFO: Added data : 0
(lldb) /bin/sh: python: command not found
Process Process-1:
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.12/3.12.7/Frameworks/Python.framework/Versions/3.12/lib/python3.12/multiprocessing/process.py", line 314, in _bootstrap
    self.run()
  File "/opt/homebrew/Cellar/python@3.12/3.12.7/Frameworks/Python.framework/Versions/3.12/lib/python3.12/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/topher/.dave/venv/lib/python3.12/site-packages/dave/server/process.py", line 104, in create_and_run
    from dave.client import DaveGUI
  File "/Users/topher/.dave/venv/lib/python3.12/site-packages/dave/client/__init__.py", line 5, in <module>
    subprocess.check_output(
  File "/opt/homebrew/Cellar/python@3.12/3.12.7/Frameworks/Python.framework/Versions/3.12/lib/python3.12/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/homebrew/Cellar/python@3.12/3.12.7/Frameworks/Python.framework/Versions/3.12/lib/python3.12/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command 'python -c "import os,sys;print(os.linesep.join(sys.path).strip())"' returned non-zero exit status 127.
```

**Comment by cjappl** [comment]
```
> which python
>
```

There is no binary called python on my system, I believe OSX (new versions, at least) only have python3:

```
> ls /usr/bin/python3
/usr/bin/python3
```

**Comment by maxmarsc** [comment]
Good catch thanks ! That's some leftover code I though I cleaned up :sweat_smile: 

This should be fixed in `v0.3.1`. Do not use `dave update` though, uninstall dave then reinstall from main (this should not be necessary in the future).

Please let me know if that fixed the issue :)

**Comment by cjappl** [comment]
Good news:This fixes it!! 
Bad news: But uncovers another issue.

I'm going to file a new one for that just to keep things atomic and not have a "mega issue". Thanks for the quick fix :)
